blogposts/2021/dev-tool-time-paul-jolly.md

```diff
@@ -0,0 +1,84 @@
+---
+title: '“It’s small things that, when combined, make your development experience that much more pleasant”: Dev Tool Time with Paul Jolly'
+description: 'Paul Jolly, CTO and founder at myitcv.io, shares how a minimalist desk setup, combined with the sophisticated programming language CUE, creates a streamlined development environment.'
```

Comment:
@nickmyyz @csbailey5t Double checking his title. We have CTO and founder at myitcv.io? He didn't add anything about myitcv in his bio, so I'm questioning this. Bio he provided us: Paul Jolly is a maintainer on the [CUE project](https://cuelang.org/). He co-created [Play with Go](https://play-with-go.dev/), contributes to the [Go project](https://golang.org/), created [govim](https://github.com/govim/govim), and helps maintains a number of other open source projects.

Reply:
@jennybergen I went by his [LinkedIn](https://uk.linkedin.com/in/pauljolly) but maybe we should stick closer to what he provided?

Reply:
The info re: myitcv.io is accurate, but I think it's preferable to go with what he sent and the emphasis on his Go work, which is what he's known for.

Reply:
@csbailey5t is this accurate?
Paul Jolly, CUE project maintainer and Play with Go co-creator
or
Paul Jolly, maintainer on the CUE project and co-creator of Play with Go

Reply:
Actually, on the original DTT LP image, we had "Maintainer of CUE, Creator of govim"

Reply:
I'd go with that last one